testimonials/testimonials.json

```diff
@@ -10,7 +10,19 @@
         "x": "egyboy_",
         "bsky": "egyboyy.bsky.social",
         "github": "egyboyy"
+      },
+    {
```

Comment:
Your formatting is wrong. Please check that with some json format checker ([this one](https://jsonformatter.curiousconcept.com) for example)